### PR TITLE
adding matthew's talk to the events - because why not :)

### PR DESCRIPTION
**Pull request by ghost**
image

### CODE REVIEW

File: _events/2017-01-02-matthew-at-agileonthebench.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+author: 
+ - Matthew Butt 
+speaker: Matthew Butt
+layout: event
+name: matthew-at-agile-on-the-bench 
+location: "Arnold Circus, London E2 7JS, UK"
+image:
+    src: /assets/img/custom/events/agile-on-the-bench.png
```
Comment (cyplo) [inline _events/2017-01-02-matthew-at-agileonthebench.md]: Do we have the rights to publish this picture ?

Comment (bnathyuw) [reply _events/2017-01-02-matthew-at-agileonthebench.md]: Good question. I'm happy to contact the organisers and ask. Shall I check about this specific image, or ask if they have an image they would be happy for us to use?

Comment (bnathyuw) [reply _events/2017-01-02-matthew-at-agileonthebench.md]: Also, I think we need to crop whatever image we use, as it's been squashed horizontally to fit the page.

Comment (ghost) [reply _events/2017-01-02-matthew-at-agileonthebench.md]: I asked on twitter, waiting for reply

**Comment by ghost** [comment]
I changed the image to a free to reuse one

**Pull request merged**